core/src/test/java/org/kohsuke/stapler/framework/io/LargeTextTest.java

```diff
@@ -26,17 +26,29 @@
 
 package org.kohsuke.stapler.framework.io;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
 import java.io.ByteArrayOutputStream;
 import java.io.EOFException;
+import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import com.google.common.base.Strings;
+import org.apache.commons.io.output.NullOutputStream;
+import org.junit.Ignore;
 import org.junit.Test;
-import static org.junit.Assert.*;
 import org.jvnet.hudson.test.Issue;
 
 public class LargeTextTest {
 
     @Issue("JENKINS-37664")
     @Test
+    @Ignore
```

Comment:
Wait, why does this existing test need to be ignored?

Reply:
ah, sorry, my mistake. fixed